src/TrailingWhitespaceMarker.cpp

```diff
@@ -19,7 +19,8 @@
 
 #include <QString>
 #include <QLineEdit>
-#include <QDebug>
+#include <SingleLineTextEdit.h>
+
 void unmarkQString(QString* text) {
     QChar middleDot(0x00B7);
     text->replace(middleDot, ' ');
```

Comment:
**:earth_africa: I18n** - so as to clearly differentiate between texts that need translation (inside `QObject::tr(...)`) and those that do not (inside our `qsl(...)` which is an "alias" of `QStringLiteral(...)`) we do not like to see `QString`s and `QChars` initialised from raw C string literals; for `QChar`s that are not going to be translated you can use `QChar::SpecialCharacter`s for some, e.g.:
```suggestion
    text->replace(middleDot, QChar::Space);
```

In other places where there isn't a **[`enum QChar::SpecialCharacter`](https://doc.qt.io/qt-6/qchar.html#SpecialCharacter-enum)** you should wrap single characters inside a `QLatin1Char(...)` wrapper instead...

Reply:
that is not my code (was committed last year: [commit 801d5e9](https://github.com/Mudlet/Mudlet/commit/801d5e9cc7fd80811c65bad98e88aaebfa9273f0)
but I will commit that change as suggested